packages/@webex/plugin-meetings/src/members/util.ts

```diff
@@ -174,6 +175,18 @@ MembersUtil.generateLowerAllHandsMemberOptions = (requestingParticipantId, locus
   locusUrl,
 });
 
+MembersUtil.generateEditDisplayNameMemberOptions = (
+  memberId,
+  requestingParticipantId,
+  alias,
+  locusUrl
+) => ({
+  memberId,
+  requestingParticipantId,
+  alias,
+  locusUrl,
+});
```

Comment:
non-blocking request

This appears to convert the spread parameters into an Object. Is there a way we can easily convert the entire stack to use the object instead?

If not, this will be fine.